src/components/layout/productCard/ProductCard.jsx

```diff
@@ -20,51 +21,35 @@ export default function ProductCard(props) {
     autoplaySpeed: 5000,
   };
 
-  const likeBtnHandler = () => {
-    if (!isLiked) {
-      return (
-        <img
-          onClick={() => setIsLiked(!isLiked)}
-          src={heartBroken}
-          className="likeIcon"
-          alt="heart broken"
-        />
-      );
-    } else {
-      return (
-        <img
-          onClick={() => setIsLiked(!isLiked)}
-          src={heart}
-          className="likeIcon"
-          alt="heart"
-        />
-      );
-    }
-  };
-
-  const sliderHandler = () => {
-    return (
-      <Slider {...settings}>
-        <div>
-          <img src={props.info.images[0]} alt="image" />
-        </div>
-        <div>
-          <img src={props.info.images[1]} alt="image" />
-        </div>
-        <div>
-          <img src={props.info.images[2]} alt="image" />
-        </div>
-      </Slider>
-    );
-  };
-
   return (
     <React.Fragment>
-      <Col xs={8} md={5} lg={3} className="productCard">
-        {/*doc-id = {props.productId[index]}*/}
-        {likeBtnHandler()}
-        {sliderHandler()}
-
+      <Col xs={8} md={5} lg={3} className="productCard"> {/*when we get the real product we need to add an id to each product*/}
+        {isLiked ? (
```

Comment:
so for these you should actually include them outside as variables, it makes the code cleaner. but i was saying that they should not be functions, they are simply variables.

```const slider = {
  return()
}
```
(in your old code you had `slider = () =>`